res/style/style.css

```diff
@@ -0,0 +1,8 @@
+:global(.placeholdertextloader){
```

Comment:
Why are we using :global here... there is no theming library here, shouldn’t this only be:
`.placeholdertextloader {
...
}`